ecllibrary/teststd/OpenSSL/TestOpenSSL.ecl

```diff
@@ -0,0 +1,246 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2025 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the License);
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an AS IS BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+Import Std;
+
+EXPORT TestOpenSSL := MODULE
+
+    EXPORT STRING PLAINTEXT := 'Colorless green ideas sleep furiously.';
+    EXPORT STRING CIPHERS_CIPHER := 'aes-256-cbc';
+    EXPORT STRING PASSPHRASE := 'mypassphrase';
+
+    EXPORT STRING RSA_PUBLIC_1 :=
+    '-----BEGIN RSA PUBLIC KEY-----' + '\n' +
+    'MIICCgKCAgEAuLYYTmPMzp13999s7bUAaJZVON+9k/2PDBF5AfHtQ40FmxwRdG3d' + '\n' +
```

Comment:
This may trigger warnings from scanning rules about private keys being exported in public repositories....  I'm not quite sure what the best solution idea is.